docs/theming.md

```diff
@@ -430,37 +430,35 @@ Since our cards only contain text and no components, let's keep it simple for no
 > Note: in the future we plan to provide a Javascript utility method for changing all derived colors and making this
 use-case easier.
 
+## Custom Sass Mixins
 
-## Dark Themes
+We also have provided a way to customize on a per component basis. We realize that you may want a button with a
+blue background and another with a red background on the same page. Each component's package is documented with all
+our provided sass mixins and what arguments to pass them. For example, here is a link to our [Button Sass Mixins docs](https://github.com/material-components/material-components-web/tree/master/packages/mdc-button#advanced-sass-mixins).
 
-Beyond what we've covered in this document so far, there's also the concept of a _dark theme_. All MDC Web components have
-been designed to work with both light themes (that assume a light-colored background) and dark themes (with dark-colored
-backgrounds), but the default is always light.
+If we continue the example stated above, here is what the sass would like:
```

Comment:
Rephrase to "Here is an example of how we could add styles to change the color of some buttons:"

(For some reason when I read "the example stated above", I was thinking it was referring to further above this section of the document, rather than just a paragraph ago)